<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MemoryWindow</class>
 <widget class="QMainWindow" name="MemoryWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Memory Configuration</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(20, 200, 220);</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1,1">
       <item>
        <widget class="QLabel" name="Tittle">
         <property name="styleSheet">
          <string notr="true">font: 75 20pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>Simulador de Procesos</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_initial_processes">
            <property name="text">
             <string>Cantidad de Procesos Iniciales:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinbox_initial_processes">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_secondary_memory">
            <property name="text">
             <string>Cantidad de Memoria Secundaria:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinbox_secondary_memory">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);</string>
            </property>
            <property name="minimum">
             <number>5</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_main_memory">
            <property name="text">
             <string>Cantidad de Memoria Principal:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinbox_main_memory">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);</string>
            </property>
            <property name="minimum">
             <number>5</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QPushButton" name="button_validate">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(31, 15, 255);</string>
            </property>
            <property name="text">
             <string>Validar</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
